pkgs/development/libraries/libucl/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, fetchFromGitHub
+, pkgconfig
```

Comment:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias.

Reply:
Noted, fixing.

Side question, where can I find these sort of informations in order to keep myself up to date?

Reply:
At the moment mostly by following nixpkgs pull requests. I am working on a tool and a docs that will point out common style issues so hopefully it will be easier in the future.